tests/test_admin_ui.py

```diff
@@ -0,0 +1,71 @@
+import os
```

Comment:
To keep with the test file naming structure we have been using in our tests this file should probably live here instead: `tests/admin/test_config.py`

Reply:
Yeah, that makes sense.